cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -289,20 +290,27 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
     }
 
     private fun runExperimental(config: OrtConfiguration): OrtResult {
-        // TODO: The experimental scanner supports using multiple scanner wrappers at once, for now use only one to stay
-        //       compatible with the existing scanner command. Once the experimental flag is removed this command can
-        //       support multiple scanners and use the proper ScannerWrapperFactories.
-        val scannerWrapper = when (scannerFactory.scannerName) {
-            "Askalono" -> Askalono.Factory().create(config.scanner, config.downloader)
-            "BoyterLc" -> BoyterLc.Factory().create(config.scanner, config.downloader)
-            "Licensee" -> Licensee.Factory().create(config.scanner, config.downloader)
-            "ScanCode" -> ScanCode.Factory().create(config.scanner, config.downloader)
-            else -> {
-                throw IllegalArgumentException(
-                    "The scanner ${scannerFactory.scannerName} is not supported by the experimental scanner."
-                )
+        // TODO: The experimental scanner supports using multiple scanner wrappers for projects and packages at once,
```

Comment:
commit message: Did you mean "To support that" instead of "To support this"

Reply:
I have rephrased the sentence to be more clear.